app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,22 @@
 <resources>
     <string name="app_name">Quiz App - Space Intern</string>
+    <string name="my_first_quiz">ჩემი პირველი ქვიზი</string>
+    <string name="enter_your_name">შეიყვანე სახელი</string>
+    <string name="start_quiz">ქვიზის დაწყება</string>
+    <string name="welcome_user">გამარჯობა, user</string>
```

Comment:
გამარჯობა, user -> იუზერს როგორც სტრინგის ნაწილს ისე აღიქვამს, დაწერე ისე, რომ ცალკე ცვლადად აღიქვამდეს. გამოიყენე სტრინგის ფორმატირება. გადახედე სხვა ფულ რექუესტებს, მსგავსი კომენტარები მაქვს

Reply:
changed in a later pull request